.github/drafts/17818/HubSpot-CRM-API.md

```diff
@@ -0,0 +1,136 @@
+# Settings User Provisioning
+
+This document will explain step by step how to create and run all available HubSpot queries for HubSpot CRM.
+
+For more information, consult official HubSpot API docs >> [HubSpot Docs](https://developers.hubspot.com/docs/api/crm/understanding-the-crm)
+
+HubSpot official docs for final users >> [HubSpot Docs](https://developers.hubspot.com/docs/cms/data/crm-objects)
+
+## Table of Contents
+
+1. [BUG Fixed][List Objects](/HubSpotCRM#list-objects)
```

Comment:
Sorry I couldn't understand this ? What is the meaning of `Bug Fixed` , `Read Object` etc here ? - would it show up like this on the documentation page ?